sonarts-sq-plugin/sonar-typescript-plugin/src/main/resources/org/sonar/l10n/typescript/rules/typescript/S1481.json

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "Unused locals should be removed",
```

Comment:
I am not sure what "locals" means here, perhaps replace it with "Unused declarations...."

Reply:
@saberduck I still think that "locals" is better (it's even used in TS doc https://www.typescriptlang.org/docs/handbook/compiler-options.html)
![image](https://user-images.githubusercontent.com/10108186/36262532-011de498-1268-11e8-9d61-3a088ef4038e.png)